keyboards/hazel/chaz/rp2040ce/info.json

```diff
@@ -0,0 +1,65 @@
+{
+    "manufacturer": "Hazel",
+    "keyboard_name": "Chaz",
+    "maintainer": "jasonhazel",
+    "url": "https://github.com/hazels-garage/chaz",
```

Comment:
`manufacturer`, `keyboard_name`, `maintainer`, and `url` lines are already specified in `info.json` in directory above, so they can be removed from this file.